catt/controllers.py

```diff
@@ -410,50 +448,32 @@ def _is_idle(self):
         # so we maintain a list of those apps.
         return status.player_state in ["UNKNOWN", "IDLE"] and self._cast.app_id not in NO_PLAYER_STATE_IDS
 
-    def _prep_app(self):
-        """Make sure desired chromecast app is running."""
-
-        if not self._cast_listener.app_ready.is_set():
-            self._cast.start_app(self._cast_listener.app_id)
-            self._cast_listener.app_ready.wait()
-
-    def _prep_control(self):
-        """Make sure chromecast is not inactive or idle."""
-
-        self._check_inactive()
-        self._cast.media_controller.block_until_active(1.0)
-        if self._is_idle:
-            raise CattCastError("Nothing is currently playing.")
-
-    def _prep_info(self):
-        """Make sure chromecast is not inactive."""
+    def volume(self, level):
```

Comment:
These are a bit misnamed, we should rename them to `set_volume`, `volume_up`, `volume_down` at some point.

Reply:
Noted